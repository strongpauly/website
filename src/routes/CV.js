import React, { Component } from "react";
import { HiFrame } from "../components/HiFrame";

export class CV extends Component {
  render() {
    return (
      <div id="cvContainer">
        <p>
          CV implemented using react, react-bootstrap and{" "}
          <a href="https://jsonresume.org/schema/">jsonresume</a>. It&apos;s
          source can be found{" "}
          <a href="https://www.github.com/strongpauly/cv">here</a>.
        </p>
        <HiFrame src="/cv/index.html" title="cv"></HiFrame>
      </div>
    );
  }
}
