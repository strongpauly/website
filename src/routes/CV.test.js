import React from 'react';
import {CV} from './CV';
import {shallow} from 'enzyme';

/* eslint-env jest */

describe('CV', () => {
  it('renders without crashing', () => {
    const app = shallow(<CV/>);
    expect(app).toMatchSnapshot();
  });

  it('creates a #cvContainer', () => {
    const app = shallow(<CV/>);
    expect(app.find('#cvContainer')).toHaveLength(1);
  });
});
