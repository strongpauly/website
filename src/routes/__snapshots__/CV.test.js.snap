// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CV renders without crashing 1`] = `
<div
  id="cvContainer"
>
  <p>
    CV implemented using react, react-bootstrap and
     
    <a
      href="https://jsonresume.org/schema/"
    >
      jsonresume
    </a>
    . It's source can be found
     
    <a
      href="https://www.github.com/strongpauly/cv"
    >
      here
    </a>
    .
  </p>
  <HiFrame
    src="/cv/index.html"
    title="cv"
  />
</div>
`;
